source/exegol-wrapper/start.rst

```diff
@@ -46,6 +46,8 @@ Many options exist to customize the creation of exegol container.
 ``-fs``, ``--update-fs``                                    Modifies the permissions of folders and sub-folders shared in your workspace to access the files created within the container using your host user account. (default: Disabled)
 ``-V VOLUMES``, ``--volume VOLUMES``                        Share a new volume between host and exegol (format: --volume /path/on/host/:/path/in/container/).
 ``-p PORTS``, ``--port PORTS``                              Share a network port between host and exegol (format: --port [<host_ipv4>:]<host_port>[:<container_port>][:<protocol>]. This configuration will disable the shared network with the host.
+``--hostname HOSTNAME``                                     Set a custom hostname to the exegol container (default: exegol-<name>)
```

Comment:
The features is not yet deployed, might be a good idea to split the dev branch with dev-img and dev-wrp ? To follow the life cycle of each project

Reply:
Deal 🤝